### DISCUSSION TITLE
Is there a way to clear my repository storage use?

### DISCUSSION THREAD

**Discussion by hugodipaolo** [community/community#39237] [New to GitHub]
Hello, I have a README profile repo that is automated, I'm worried everything is being stored and the repo storage is about 72Gb.
I would like to know if there is any way to clear some spaces to make sure i will not exceed the storage limit on the repo.

Those files are part of files from my repository's history. Can I just clear this history to delete the files ?
Many Thanks.

**Comment by Preetiraj3697** [comment]
To delete an owned storage repository:

Make sure that all content has been removed from the storage repository you wish to delete.

Select the repository in the table and click Delete.

Confirm that you wish to delete this storage repository: click OK to continue.

Click OK to execute the delete operation.

To delete a storage repository without removing the contents:

Select the storage repository in the table in the Repositories tab. Click Present-Unpresent and unpresent the storage repository from all servers. Click OK.

Select the repository in the table and click Edit. Select the Release Ownership check box and click OK.

Select the now unowned repository in the table and click Delete.

Confirm that you wish to delete this storage repository: click OK to continue.

Click OK to execute the delete operation.
